pkg/config/config.go

```diff
@@ -24,6 +29,10 @@ func Load(config *api.Config) error {
 	if err != nil {
 		return errors.Trace(fmt.Errorf("Error unmarshalling config file: %w", err))
 	}
+	if config.Target.RepoName == "" {
+		klog.V(2).Infof("'target.repoName' property is empty. Using 'myrepo' default value")
```

Comment:
this should be either a warning or a bigger verbosity message

Reply:
I think I prefer a warning